interface/backend/api/views.py

```diff
@@ -42,6 +43,27 @@ class ImageSeriesViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.ImageSeriesSerializer
 
 
+class ImageMetadataApiView(APIView):   
+
+    def get(self, request):
+        '''
+        Get metadata of a DICOM image including the image in base64 format.
+        Example: .../api/images/metadata?dicom_location='FULL_PATH_TO_IMAGE'
+        ---
+        parameters:
+            - name: dicom_location
+            description: 'QUOTED' full location of the image
+            required: true
+            type: string
+        '''
+        path = request.GET.get('dicom_location')
+        if path is None:
+            raise Exception('dicom_location not provided')
+        path = path[1:-1]   # un-quoting the string
```

Comment:
Hang on, why would we be quoted?

Reply:
(Please clarify in docstring for posterity, not here....)

Reply:
@lamby  unquoted path seems to create problem with existing url patterns.

Reply:
"seems to"?

And again, please clarify the reason in the docstring itself, otherwise these important comments just get lost :)